esrally/track/loader.py

```diff
@@ -723,6 +733,7 @@ def filter_out_match(task, filters, exclude):
 
     return t
 
+
```

Comment:
This change looks unrelated to the PR. Can we revert it?

Reply:
I think we should keep this one. There should be two spaces here.